met-web/src/components/survey/building/index.tsx

```diff
@@ -99,7 +99,9 @@ const SurveyFormBuilder = () => {
             dispatch(
                 openNotification({
                     severity: 'success',
-                    text: 'The survey was successfully built',
+                    text: savedSurvey.engagement?.id
```

Comment:
This will create an extra long notification, I think we decided that toast notifications should be small and simple

Reply:
Might have to discuss with marie anik then she mentioned that the notification was not specific enough for the user its still the same when you build a survey but when you create a survey and link it will mention that the survey is linked

Reply:
Yeah maybe she'd be content with 'Survey was successfully added to engagement' without the names

Reply:
I talked to Marie Anik perfect well go with that then